fanciblock.box.admin.inc

```diff
@@ -305,11 +305,11 @@ function fanciblock_box_delete_box_submit($form, &$form_state) {
   $grid = $fanciblock['settings']['grid']['grid']--;
   // Do the delete.
   unset($fanciblock['boxes'][$form['#boxid']]);
-  config_set('fanciblock.settings', $form['#fanciblock_name'], $fanciblock); 
-
+  fanciblock_box_save($fanciblock);
   $t_args = array('%title' => $deleted);
   backdrop_set_message(t('The fanciblock box %title has been deleted.', $t_args));
   watchdog('fanciblock', 'Deleted fanciblock box %title.', $t_args, WATCHDOG_NOTICE);
+  backdrop_goto('admin/config/media/fanciblocks/' . $form['#fanciblock_name']);
```

Comment:
Added this because it was staying on confirmation page after "confirmed". Causing the user to keep confirming the same thing over and over.